internal/application/application.go

```diff
@@ -317,6 +311,12 @@ func Unstage(ctx context.Context, cluster *kubernetes.Cluster, appRef models.App
 		if err != nil {
 			return err
 		}
+
+		// TODO: write a test for this?
+		// TODO: Don't fail if the file is already gone?
```

Comment:
Hmm, I'm not sure we need a test for this. The underlying library will give a 204 if the object is missing (so no error): https://github.com/minio/minio-go/blob/master/api-remove.go#L186-L189

Should we skip this test?

Reply:
I vote skip :)

Reply:
Skipping.